apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -29,6 +31,8 @@ export default function ListBoxPortal({ app, fieldIdentifier, stateName, element
 }
 
 export function ListBoxInline({ app, fieldIdentifier, stateName = '$', options = {} }) {
+  const [opts, setOptions] = useState({ ...options });
```

Comment:
As all options are stored as state I would test the component to make sure that it isn't rerendered unnecessary. Another approach would be to only store the configurable settings in state and take the rest directly from options ?